orbit/pkg/table/firmwarepasswd/firmwarepasswd.go

```diff
@@ -57,7 +56,7 @@ func New(logger log.Logger) *Table {
 		})
 
 	return &Table{
-		logger: level.NewFilter(logger, level.AllowInfo()),
+		logger: logger,
```

Comment:
We should probably add the table name here like in the other table loggers

Reply:
Done!